### PR TITLE
Add and use pkcs12 in tls-crypt-v2 e2e test

### PR DESCRIPTION
**Pull request by reynir**
This will exercise some of the pkcs12 code.

I also removed unused symbolic links.

### CODE REVIEW

**Comment by reynir** [comment]
I added a test with `peer-fingerprint` too, but `--peer-fingerprint` is OpenVPN>=2.6 so doesn't work with CI.

**Comment by hannesm** [comment]
I opened #273 to get openvpn 2.6 installed on the GitHub worker (following the OpenVPN instructions). I guess that would be nice for this here as well :)

File: test/e2e/miragevpn-server.sh [hunk 1]
```diff
@@ -28,9 +28,9 @@ trap cleanup EXIT
 
 (
 	sleep 1
-	openvpn --cd "$config_dir" --config "client.conf" --dev-type tun --dev "$tun_interface" --writepid "$pidfile" --script-security 2 --up ../client-up.sh > /dev/null
+	openvpn --cd "$config_dir" --config "client.conf" --dev-type tun --dev "$tun_interface" --writepid "$pidfile" --script-security 2 --up ../client-up.sh
```
Comment (hannesm) [inline test/e2e/miragevpn-server.sh:31]: I'm undecided about the output here.. somehow it is nice to spot the issues easily in CI, somehow it is also nice to not get a lot of output.

In the end, I'm fine without the redirection to /dev/null. if there's need to see less output, someone invoking the scripts can still redirect to /dev/null. :)

**Comment by hannesm** [comment]
looks good to me

File: test/e2e/miragevpn-server.sh (hunk 1 above)
Comment (reynir) [reply test/e2e/miragevpn-server.sh:31]: I removed the redirection to /dev/null to debug in CI. We should add it back, I just forgot.

**Comment by reynir** [comment]
The two numbers are unfortunately not 100% comparable as for the 2nd numbers I rebased on #273 which covers more code as well. While rebasing I also dropped the "debug" commit which removed the redirection to /dev/null.

**Comment by hannesm** [comment]
The other PR was at 72.76%, here we're nearly 1% more.

**Comment by reynir** [comment]
I squashed the fixup commit before merging

**Comment by hannesm** [comment]
Should we remove the redirects? As mentioned above, fine with me.

**Pull request merged**